source/Halibut/ServiceModel/PendingRequestQueueInternalException.cs

```diff
@@ -0,0 +1,11 @@
+using System;
+
+namespace Halibut.ServiceModel
+{
+    public class PendingRequestQueueInternalException : Exception 
```

Comment:
Do we not have something similar already?

Also, are we planning on catching this type in Tentacle Client/Octopus Server?

Reply:
I didn't see something like this already existing.

I don't plan on catching it yet, but I do want a specific exception type to capture what is happening. Rather than always doing string matching.